src/Tribe/Main.php

```diff
@@ -944,7 +944,7 @@ public function init() {
 			$this->singular_event_label_lowercase             = tribe_get_event_label_singular_lowercase();
 			$this->plural_event_label_lowercase               = tribe_get_event_label_plural_lowercase();
 
-			$this->post_type_args['rewrite']['slug']            = $rewrite->prepare_slug( $this->rewriteSlugSingular, self::POSTTYPE, false );
+			$this->post_type_args['rewrite']['slug']          = $rewrite->prepare_slug( $this->rewriteSlugSingular, self::POSTTYPE, false );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Variable "post_type_args" is not in valid camel caps format
* :no_entry_sign: Equals sign not aligned correctly; expected 1 space but found 10 spaces
* :no_entry_sign: Object property "$rewriteSlugSingular" is not in valid snake_case format, try "$rewrite_slug_singular"
* :warning: Line exceeds 120 characters; contains 140 characters

<sub>hash: 11418f02ecd4d0382d0d3f9be7dc21bc</sub>

Reply:
:fail: